package.json

```diff
@@ -41,6 +47,7 @@
     "ember-common-tags": "0.1.2",
     "ember-disable-prototype-extensions": "^1.1.2",
     "ember-export-application-global": "^2.0.0",
+    "ember-fetch": "3.3.1",
```

Comment:
this replaces the deprecated `ember-network`, so we should remove that dependency. There's a part of the tests in [`module-for-acceptance.js`](https://github.com/bgentry/ember-apollo-client/blob/178dbb7f836b26efcfcf735e0e71520e5630474d/tests/helpers/module-for-acceptance.js#L20) that needs to be updated to account for that.

Reply:
My PR, solve this #59 when it is merged